js/05/main.js

```diff
@@ -0,0 +1,47 @@
+// First remove all the punctuations and change the string to array and count the number of words in the array
+let text = 'I love teaching and empowering people. I teach HTML, CSS, JS, React, Python.';
+const words = text.split(' ');
+console.log(words);
+console.log(words.length);
+
+//In countries array check if 'Ethiopia' exists in the array if it exists print 'ETHIOPIA'. If it does not exist add to the countries list and print the array.
+let ethiopia = 'Ethiopia';
+if (countries.includes(ethiopia)) {
```

Comment:
We can simplify this code via ternary operator - 
`countries.includes(ethiopia) ? console.log('ETHIPIA) : countries.push(ethiopia)`